### DISCUSSION TITLE
Support the deployment of object detection algorithm EfficientDet on TensorRT backend 支持目标检测模型EfficientDet在TensorRT侧的部署

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#637] [MMDeploy]
# Support the deployment of object detection algorithm EfficientDet on TensorRT backend 
 支持目标检测模型EfficientDet在TensorRT侧的部署 

Support the deployment of EfficientDet algorithm from mmdetecion onTensorRT.
You can refer to the following resources:
1. [EfficientDet]([https://github.com/open-mmlab/mmdetection/blob/main/projects/EfficientDet/README.md)](https://github.com/open-mmlab/mmdetection/blob/main/projects/EfficientDet/README.md)) 
2. [MMDet Deployment]([https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet.html](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet.html) )
3. [Deployment Tutorials]([https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.html](https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.html) )

在mmdeploy中支持EfficientDet 模型在TensorRT侧部署
相关资源：
1. [EfficientDet]([https://github.com/open-mmlab/mmdetection/blob/main/projects/EfficientDet/README.md)](https://github.com/open-mmlab/mmdetection/blob/main/projects/EfficientDet/README.md)) 
2. [MMdet部署]([https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet.html](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet.html) )
3. [模型部署教程]([https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.html](https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.html) )

## Task Information

**Repository**：MMDeploy

**Difficulty**：Medium

**Points**：40

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：xue-yun-liang

**Claim Time**：2023年10月24日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by Ryosnow** [comment]
Claim the mission
